### PR TITLE
Mention that we don't change the hash function

### PR DESCRIPTION
**Pull request by real-or-random**
recycled from #62

### CODE REVIEW

File: bip-schnorr.mediawiki [hunk 1]
```diff
@@ -21,7 +21,7 @@ This document is licensed under the 2-clause BSD license.
 === Motivation ===
 
 Bitcoin has traditionally used
-[https://en.wikipedia.org/wiki/Elliptic_Curve_Digital_Signature_Algorithm ECDSA] signatures over the [https://www.secg.org/sec2-v2.pdf secp256k1 curve] for authenticating
+[https://en.wikipedia.org/wiki/Elliptic_Curve_Digital_Signature_Algorithm ECDSA] signatures over the [https://www.secg.org/sec2-v2.pdf secp256k1 curve] and with [https://en.wikipedia.org/wiki/SHA-2 SHA256] hashes for authenticating
```
Comment (jonasnick) [inline bip-schnorr.mediawiki]: Seems like either "and" or "with" need to go.

File: bip-schnorr.mediawiki [hunk 2]
```diff
@@ -38,7 +38,7 @@ made:
 * '''Public key encoding''': Instead of using ''compressed'' 33-byte encodings of elliptic curve points which are common in Bitcoin today, public keys in this proposal are encoded as 32 bytes.
 * '''Batch verification''': The specific formulation of ECDSA signatures that is standardized cannot be verified more efficiently in batch compared to individually, unless additional witness data is added. Changing the signature scheme offers an opportunity to address this.
 
-By reusing the same curve as Bitcoin uses for ECDSA, we are able to retain existing mechanisms for choosing secret and public keys, and we avoid introducing new assumptions about elliptic curve group security.
+By reusing the same curve and hash function as Bitcoin has used for ECDSA, we are able to retain existing mechanisms for choosing private and public keys, and we avoid introducing new assumptions about the security of elliptic curves and hash functions.
```
Comment (jonasnick) [inline bip-schnorr.mediawiki]: "has used" sounds like it's not used anymore?

(I thought that we had decided on secret vs. private key in bip-schnorr, but actually we only did that for bip-taproot/tapscript. I'm in the "secret" key camp fwiw).

Comment (jonasnick) [reply bip-schnorr.mediawiki]: Opened PR to replace private keys with secret keys #180

Comment (real-or-random) [reply bip-schnorr.mediawiki]: > "has used" sounds like it's not used anymore?

It's present perfect, so it should not indicate that it's not used anymore. But I can revert it, yes.

File: bip-schnorr.mediawiki (hunk 1 above)
Comment (real-or-random) [reply bip-schnorr.mediawiki]: Yeah, it was on purpose but maybe removing the "and" is better.

Comment (real-or-random) [reply bip-schnorr.mediawiki]:
```suggestion
[https://en.wikipedia.org/wiki/Elliptic_Curve_Digital_Signature_Algorithm ECDSA] signatures over the [https://www.secg.org/sec2-v2.pdf secp256k1 curve] with [https://en.wikipedia.org/wiki/SHA-2 SHA256] hashes for authenticating
```

File: bip-schnorr.mediawiki (hunk 2 above)
Comment (real-or-random) [reply bip-schnorr.mediawiki]: I can still revert it, of course.

Comment (real-or-random) [inline bip-schnorr.mediawiki]:
```suggestion
By reusing the same curve and hash function as Bitcoin uses for ECDSA, we are able to retain existing mechanisms for choosing private and public keys, and we avoid introducing new assumptions about the security of elliptic curves and hash functions.
```

File: bip-schnorr.mediawiki [hunk 3]
```diff
@@ -38,7 +38,7 @@ made:
 * '''Public key encoding''': Instead of using ''compressed'' 33-byte encodings of elliptic curve points which are common in Bitcoin today, public keys in this proposal are encoded as 32 bytes.
 * '''Batch verification''': The specific formulation of ECDSA signatures that is standardized cannot be verified more efficiently in batch compared to individually, unless additional witness data is added. Changing the signature scheme offers an opportunity to address this.
 
-By reusing the same curve as Bitcoin uses for ECDSA, we are able to retain existing mechanisms for choosing secret and public keys, and we avoid introducing new assumptions about elliptic curve group security.
+By reusing the same curve and hash function as Bitcoin uses for ECDSA, we are able to retain existing mechanisms for choosing private and public keys, and we avoid introducing new assumptions about the security of elliptic curves and hash functions.
```
Comment (sipa) [inline bip-schnorr.mediawiki]: This reverts #180 now.

**Comment by real-or-random** [comment]
changed to secret key

**Pull request merged**